src/util/ellipsizeUrl.js

```diff
@@ -0,0 +1,8 @@
+import {
+    URL_MAX_CHARS,
+  } from '../constants.js'
+import { isURL } from '../util/isURL'
+
+export const ellipsizeUrl = value => {
+    return (value && isURL) ? value.substring(0, URL_MAX_CHARS) + '...' : value
```

Comment:
Parentheses are redundant since order-of-operations of ternary is lower than `+`.